Source/ExcelDna.IntelliSense/IntelliSenseServer.cs

```diff
@@ -30,7 +30,7 @@ namespace ExcelDna.IntelliSense
     // REMEMBER: COM events are not necessarily safe macro contexts.
     public static class IntelliSenseServer
     {
-        const string ServerVersion = "1.0.10";  // TODO: Define and manage this somewhere else
+        const string ServerVersion = "1.2.0";  // TODO: Define and manage this somewhere else
```

Comment:
Why so high? Maybe 1.1.0 for the next version (and 1.1.1 etc. for small updates during the 1.1.0 ExcelDna.Integration life?)

Reply:
I'm not sure the ServerVersion and the AssemblyVersion can be kept in sync in the strong-name world? Then we might as well keep the assembly version fixed at 1.1? (or 1.1.1). I'm getting confused...

Reply:
> Why so high? Maybe 1.1.0 for the next version (and 1.1.1 etc. for small updates during the 1.1.0 ExcelDna.Integration life?)

`AssemblyVersion` is `1.1.0` already. It seems this `ServerVersion` variable went out of sync with the `AssemblyVersion`. So in this PR I'm bumping the `AssemblyVersion` to `1.2.0` and keeping `ServerVersion` in sync...

Reply:
> I'm not sure the ServerVersion and the AssemblyVersion can be kept in sync in the strong-name world? Then we might as well keep the assembly version fixed at 1.1? (or 1.1.1). I'm getting confused...

I'd suggest we sync `ServerVersion`  with `AssemblyInformationalVersion` (which is what gets translated to `ProductVersion` at the end.

The reason being that any modern build servers (such as Azure Pipelines) supports updating the `AssemblyInformationalVersion` during build and baking it into the compiled assembly... And to make `ServerVersion` get bumped it would take some scripting.

If I understand correctly, `ServerVersion` is just a custom metadata that is used to determine the highest version of the IntelliSense from all the available IntelliSense servers across all add-ins, so the number doesn't really matter much, as long as it's always higher than the previous release.